specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -3984,33 +4600,45 @@
             "description": "The name of the resource group."
           },
           {
-            "name": "restorePointCollectionName",
+            "name": "capacityReservationGroupName",
             "in": "path",
```

Comment:
is this a breaking change? was it approved?

Reply:
this is not breaking change, the diff is showing incorrectly that we are changing the name here. but in actual file, we have just added new sections for capacity reservation and didnt touch the existing  restorepoint api operation swagger changes,